src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
+$width-day: 100px;
+$gap: 1px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
```

Comment:
Placing both outer and inner properties in the same selector is prohibited according to BEM.